source/_components/ddwrt.markdown

```diff
@@ -50,6 +54,32 @@ verify_ssl:
   required: false
   type: boolean
   default: true
+type:
+  description: The type of devices to query from the router. (lan, wireless, dhcp)
+  required: false
+  type: string
+  default: lan
+monitored_conditions:
+  description: Wireless conditions to display in the frontend.
+  required: false
+  type: list
+  keys:
+    interface:
+      description: Display the interface name (w0, w1, w2)
+    tx_rate:
+      description: The connection speed of the downlink, in Mbps
+    rx_rate:
+      description: The connection speed of the uplink, in Mbps
+    info:
+      description: Connection type: Legacy, HT20, VHT80PS...
```

Comment:
```suggestion
      description: "Connection type: `Legacy`, `HT20`, `VHT80PS`."
```

Reply:
Oh ok thanks I asked about this but we weren't sure, I'll update now

Reply:
Oh nevermind I thought it was something else XD